src/System.IO.Pipelines/DefaultReadableBufferExtensions.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Binary;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+
+namespace System.IO.Pipelines
+{
+    /// <summary>
+    /// Common extension methods against readable buffers
+    /// </summary>
+    public static class DefaultReadableBufferExtensions
+    {
+        /// <summary>
+        /// Copies a <see cref="ReadableBuffer"/> to a <see cref="Stream"/> asynchronously
+        /// </summary>
+        /// <param name="buffer">The <see cref="ReadableBuffer"/> to copy</param>
+        /// <param name="stream">The target <see cref="Stream"/></param>
+        /// <returns></returns>
+        public static async Task CopyToAsync(this ReadableBuffer buffer, Stream stream)
+        {
+            foreach (var memory in buffer)
+            {
+                ArraySegment<byte> data;
+                if (memory.TryGetArray(out data))
+                {
+                    await stream.WriteAsync(data.Array, data.Offset, data.Count);
```

Comment:
All of the awaits on tasks should use ConfigureAwait(false)

Reply:
Is this still true?